README.md

```diff
@@ -16,20 +16,32 @@ Zeebe provides visibility into and control over business processes that span mul
 * Export process data for monitoring and analysis
 * Engage with an active community
 
-[Learn more at zeebe.io](https://zeebe.io)
+[Learn more at camunda.com](https://camunda.com/products/cloud/workflow-engine/)
+
+## Release Lifecycle
+
+Our release cadence within major releases is a minor release every three months, with an alpha release on each of the two months between minor releases. Releases happen on the second Tuesday of the month, Berlin time.
```

Comment:
>Releases happen on the second Tuesday of the month, Berlin time.

Are we sure we want to state this so concretely. Perhaps something like:

> We try to release on the second Tuesday of the month, Berlin time.

Reply:
"Do or do not, there is no try" - Yoda

My thoughts on this:

If a release is delayed, then that's a breakdown against our schedule, and we can root cause diagnose it, and play again for the next one. This gives us increased performance and reliability over time, because we have a standard and hold ourselves to it.  

If our public commitment is to "trying", then even missing the target is still "success", because we successfully "tried". 

We may not hit the target with every shot, but that doesn't mean we should back away from having a concrete target, calling our shot, taking a committed shot every time. And dealing with a hit or miss as a hit or miss. 

As long as we stay in communication when there are delays, I think it will be fine. And if we consistently or regularly miss, then we need to look at changing the declared target to something realistic and reliable. 

That's my take.

Reply:
ping @npepinpe @menski, can the team commit to this?

Reply:
I agree with Josh, let's state it (this is our official policy anyway at Camunda), and when there are delays or issues, we should clearly communicate them as soon as possible. But there's no reason to introduce doubt here.

Reply:
I would propose to write here a timezone, Berlin exist also in the US. https://en.wikipedia.org/wiki/Berlin_(disambiguation)

Reply:
```suggestion
Our release cadence within major releases is a minor release every three months, with an alpha release on each of the two months between minor releases. Releases happen on the second Tuesday of the month, Berlin time (CET).
```